cmd/cmd.go

```diff
@@ -1,29 +1,13 @@
 package cmd
 
 import (
-	"path/filepath"
-	"regexp"
-	"strings"
+	"log"
+	"os"
 )
 
-const (
-	msgPleaseAuthenticate = "You must be authenticated. Run `exercism configure --key=YOUR_API_KEY`."
-)
-
-func isTest(path string) bool {
-	ext := filepath.Ext(path)
-	if ext == ".t" {
-		return true
-	}
-
-	file := filepath.Base(path)
-	name := file[:len(file)-len(ext)]
-	if name == "test" || name == "spec" {
-		return true
-	}
-	return regexp.MustCompile(`[\._-]?([tT]est|[sS]pec)`).MatchString(name)
-}
-
-func isREADME(path string) bool {
-	return strings.Contains(path, "README")
+// Bail handles exitable errors in commands.
+// TODO: figure out what goes here.
+func Bail(err error) {
```

Comment:
That's better. 
I'm happy to refactor the code and open a PR to this branch. I know you're deep in design and build.